frontend/public/components/monitoring/alert-manager-receiver-forms.tsx

```diff
@@ -173,25 +173,21 @@ const RoutingLabels = ({ routeLabels, setRouteLabels, showAddLabel }) => {
                     />
                   </div>
                 </div>
-              </div>
-              {!isDefaultReceiverRouteLabel && (
-                <>
-                  <div className="row">
-                    <div className="col-xs-12 col-sm-12">
-                      <div className="form-group">
-                        <label className="co-no-bold">
-                          <input
-                            type="checkbox"
-                            onChange={(e) => onRoutingLabelRegexChange(e, _.toNumber(i))}
-                            checked={routeLabel.isRegex}
-                          />
-                          &nbsp; Regular Expression
-                        </label>
-                      </div>
+                {!isDefaultReceiverRouteLabel && (
+                  <div className="col-xs-4">
+                    <div className="form-group co-alert-manager-routing-labels-useregex">
+                      <label className="co-no-bold">
```

Comment:
Not new, but you shouldn't need this style if you wrap this in a `<div className="checkbox">`.